### PR TITLE
WIP: Convert to Flask (take 2)

### PR DESCRIPTION
**Pull request by bartaz**
Built on top of changes in  #2773
Rebased to include all standalone examples from master

```
./run
```

Examples listing page doesn't work yet, but examples themselves do:
https://vanilla-framework-canonical-web-and-design-pr-2779.run.demo.haus/examples/base/code

### CODE REVIEW

File: urls.txt [hunk 1]
```diff
@@ -0,0 +1,215 @@
+http://localhost:8101
```
Comment (nottrobin) [inline urls.txt]: I thought I'd added this to `.gitignore`?

Comment (bartaz) [reply urls.txt]: Maybe it was in later commit that I haven't cherry picked yet?

File: redirects.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/en/?: /
+/en/(?P<path>.*): /{path}
```
Comment (nottrobin) [inline redirects.yaml]: It occurred to me last night, we should probably add:

``` yaml
/(?P<prefix>.+)\.html: /{prefix}
```

Or something

**Comment by bartaz** [comment]
Moved to feature branch #2782

**Pull request closed without merging**